src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput/hooks.js

```diff
@@ -8,13 +8,25 @@ const useAdjustedGradeInputData = () => {
   const hintText = possibleGrade && ` ${getLocalizedSlash()} ${possibleGrade}`;
 
   const onChange = ({ target }) => {
-    setModalState({ adjustedGradeValue: target.value });
+    let adjustedGradeValue;
+    switch (true) {
+      case target.value < 0:
+        adjustedGradeValue = 0;
+        break;
+      case possibleGrade && target.value > possibleGrade:
+        adjustedGradeValue = possibleGrade;
```

Comment:
My point was that instead of artificially modifying the number the user was trying to enter, we should show an error message instead.  This is so they can understand what the problem was.

Reply:
Got it

The proposal makes sense, but I can't say when I'll have time to implement it (too loaded now with other stuff)